Day-II-Calculator/calculator/src/components/ButtonComponents/Button.css

```diff
@@ -0,0 +1,48 @@
+.largeButton {
+	height:90px;
+	width:75%;
+	background: lightgray;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	font-size: 3.0rem;
+	font-weight: normal;
+	border-bottom: 1px solid gray;
+	cursor: pointer;
+	box-sizing: border-box;
+}
+
```

Comment:
See if you can find a way to make the calculator font match a real calculator. (HINT: 14 segment display works best to show the multiplication symbol best.)